WardReports/Generate_weekly_ward_report_scripts.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+import numpy as np
+
+import datetime as dt
+import math
+
+import civis
+import os
+
+from helpers import *
+
+client = civis.APIClient()
+
+def main():
+    #actual ward table import to be used once testing is complete
+    '''
+    ward_email_data = civis.io.read_civis(database='City of Chicago',
+                                          table = 'scratch.ward_office_info',
+                                          use_pandas = True)
+                                          '''
+
+    #Generate fake table that should be replaced with actual ward table later
+    wards = list(range(1,51))
+    emails = ['srao@civisanalytics.com' for i in range(50)]
+    platform_user = ['Yes' for i in range(25)] + ['No' for i in range(25)]
+    ward_email_data = pd.DataFrame(list(zip(wards, emails, platform_user)),
+                   columns =['WARD', 'Ward_Office_Email', 'Platform User'])
+
+
+    #Pull ward aggregation and household data
+    query = """SELECT * FROM cic.ward_visualization_table;"""
+    ward_df = civis.io.read_civis_sql(query,database='City of Chicago',use_pandas = True)
+    ward_df.dropna(inplace=True)
+
+    #Calculate counted and uncounted households per ward
+    ward_df['Counted Households'] = round(ward_df['mail_return_rate_cen_2010'] * (ward_df['tot_occp_units_acs_13_17']/100))
+    ward_df['Uncounted Households'] = ward_df['tot_occp_units_acs_13_17'] - ward_df['Counted Households']
+    ward_df['Percent Counted'] = round(ward_df['Counted Households']*100/ward_df['tot_occp_units_acs_13_17'],1)
+    ward_df['Percent Uncounted'] = round(ward_df['Uncounted Households']*100/ward_df['tot_occp_units_acs_13_17'],1)
```

Comment:
now that im thikning about it, lets move these calculations to the ward visualization SQL script. no reason to have some calculations there and some here. that way gene can use them as well.